content/src/ports/activeEntities.ts

```diff
@@ -322,7 +385,27 @@ export function createActiveEntitiesComponent(
   function reset() {
     entityIdByPointers.clear()
     collectionUrnsByPrefixCache.clear()
-    cache.clear()
+    entityCache.clear()
+  }
+
+  async function initialize(database: DatabaseClient) {
+    logger.info('Initializing active entities cache', {
+      entityTypes: cachedEntityntityTypes.map((t) => t.toString()).toString()
+    })
+    try {
+      for (const entityType of cachedEntityntityTypes) {
+        await populateEntityType(database, entityType)
+      }
+      logger.info('Active entities cache initialized')
+    } catch (error) {
+      logger.error('Error initializing active entities cache', { error })
+    }
+  }
+
+  function getAllCachedScenes() {
+    return Array.from(fixedCache.values()).filter(
+      (entity): entity is Entity => typeof entity !== 'string' && entity.type === EntityType.SCENE
```

Comment:
I'm not familiar with this check, but can we prevent `typeof entity !== string` on setting time instead of retrieve time?

Reply:
The LRU cache was already with this workaround for unsetting pointers with no longer active entities. I haven't had the intention of modifying this current behavior